assets/js/app/LocusZoom.js

```diff
@@ -15,35 +15,48 @@
     // Create a new instance by instance class and attach it to a div by ID
     // NOTE: if no InstanceClass is passed then the instance will use the Intance base class.
     //       The DefaultInstance class must be passed explicitly just as any other class that extends Instance.
-    exports.addInstanceToDivById = function(InstanceClass, id){
+    exports.addInstanceToDivById = function(id, datasource, layout, state){
```

Comment:
I thought i commented the addInstanceToDivById test out of the test file in my code. When I ran it all tests passed. I think this really should be a module but being a module requires everything to be in the same file. The fact that we started with different files was probably the wrong choice and has made many things more difficult.